assets/js/app.js

```diff
@@ -18,7 +18,18 @@ import NProgress from "nprogress"
 import {LiveSocket} from "phoenix_live_view"
 
 let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute("content")
-let liveSocket = new LiveSocket("/live", Socket, {params: {_csrf_token: csrfToken}})
+
+let Hooks = {}
+Hooks.AnimationColorUpdate = {
+  mounted() {
+    this.handleEvent("ACU", colors => {
```

Comment:
That brings the message size up to ~285 bytes. I initially had it as `AnimationColorUpdate`.

Reply:
Which could be a moot point, really. It's a quibble over like... 2% difference from the original huge messages.

Reply:
If you're trying to keep the bytes down, how about a smaller name for the same thing, like `paint` or `draw`maybe? Somebody could probably understand those in the context of animations being updated. `ACU` is a lot harder to deduce.

Reply:
Also based on the image I uploaded to Slack earlier, I think we're getting to a point where reducing bytes is producing diminishing returns, and the next step is going to be to get the reductions down.

![image](https://user-images.githubusercontent.com/4755047/87738123-82640600-c791-11ea-96e2-b4c9fd0484e3.png)

Reply:
Oh yeah, `draw` works just fine. I'll use that.

As far as getting the reductions down... I'm not really sure where to start with that. I think part of it is just the way Live View works. Running the dashboard and changing tabs a lot creates a massive number of reductions in the process that handles the dashboard. Seems like there's a lot of moving parts in Live View...

Reply:
One of the first steps would be to move `Engine.get_animation_config` so that all of those processes aren't bottlenecked on the engine when that happens. Another thing to try may be to throttle the frames sent to live view to see how that impacts the system. There may be some optimizations we could do in [`matrix_live.ex`](https://github.com/ElixirSeattle/xebow/blob/master/lib/xebow_web/live/matrix_live.ex) as well.